index.js

```diff
@@ -271,7 +283,37 @@ function logError(context, error){
             return null;
         }
       } while (nbResults === pageSize);
-  
+
+    if (version >= "8.0" && !data.noSecurityHotspot) {
+      do {
```

Comment:
before do, missing `page = 1;` initialization. in my tests, page start at 2 so no hotspot issue is found